transpiler/src/zero_generator.rs

```diff
@@ -101,25 +100,10 @@ impl EvaluationStack {
         let idx = self.offset_of_last_element();
         self.state.pop();
         idx
-    }
-
-    fn add3(&mut self) -> Vec<GeneralInstruction> {
-        let n = self.state.len();
-        assert!(n > 2);
-
-        let x_addr = LocalOrImmediate::Local(self.offset_of_ith_element(n - 2));
-        let y_addr = LocalOrImmediate::Local(self.offset_of_ith_element(n - 1));
-        let z_addr = LocalOrImmediate::Local(self.offset_of_ith_element(n));
+    }*/
 
-        let dst_addr = self.offset_of_ith_element(n - 2);
-
-        let add3_inst = Instruction::Add3 {
-            x: x_addr,
-            y: y_addr,
-            z: z_addr,
-            dst: dst_addr,
-        };
-        vec![GeneralInstruction::Real(add3_inst)]
+    fn address(&mut self) -> LocalOffset {
```

Comment:
why even have this function rather than merely just outputting the constant?

Reply:
Good question haha... I was thinking that since our design is so up-in-the-air it'd be nice to have a little bit of flexibility; but I'd be fine with just using the constant

Reply:
yeah let's keep this function until we've settled on the model!